src/DDoSX/RecordClient.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client as BaseClient;
+use UKFast\SDK\DDoSX\Entities\Record;
+
+class RecordClient extends BaseClient
+{
+    protected $basePath = 'ddosx/';
+
+    /**
+     * @param $domainName
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return int|\UKFast\SDK\Page
+     */
+    public function getPage($domainName, $page = 1, $perPage = 20, $filters = [])
+    {
+        $page = $this->paginatedRequest('v1/domains/' . $domainName . '/records', $page, $perPage, $filters);
+        $page->serializeWith(function ($item) {
+            return new Record($item);
```

Comment:
I think you'll need to map the API properties to the SDK properties.

I imagine the api doesn't return `safednsRecordId` but probably returns `safedns_record_id`

You can do this with

```php
return new Record($this->apiToFriendly($item, ['safedns_record_id' => 'safednsRecordId']));
```

You can do this with